pkg/controller/operators/olm/apiservices.go

```diff
@@ -262,11 +305,6 @@ func (a *Operator) installAPIServiceRequirements(desc v1alpha1.APIServiceDescrip
 
 	existingService, err := a.lister.CoreV1().ServiceLister().Services(csv.GetNamespace()).Get(service.GetName())
 	if err == nil {
-		// Check if the only owners are this CSV or in this CSV's replacement chain
-		if !ownerutil.Adoptable(csv, existingService.GetOwnerReferences()) {
```

Comment:
why don't we need this anymore?

Reply:
This must have been removed while I was testing. Will put back soon.